gcp/table_gcp_kubernetes_cluster.go

```diff
@@ -0,0 +1,424 @@
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "server_response",
+				Description: "ServerResponse contains the HTTP response code and headers from the server.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("googleapi.ServerResponse"),
+			},
+
+			// standard steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromP(gcpKubernetesClusterTurbotData, "Akas"),
+			},
+
+			// standard gcp columns
+			{
+				Name:        "location",
+				Description: ColumnDescriptionLocation,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "project",
+				Description: ColumnDescriptionProject,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(gcpKubernetesClusterTurbotData, "Project"),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listKubernetesClusters(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("listKubernetesClusters")
+
+	// Create Service Connection
+	service, err := ContainerService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+
+	resp, err := service.Projects.Locations.Clusters.List("projects/" + project + "/locations/-").Do()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cluster := range resp.Clusters {
+		d.StreamListItem(ctx, cluster)
+	}
+
+	return nil, nil
+}
+
+func getKubernetesCluster(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getKubernetesCluster")
+
+	// Create Service Connection
+	service, err := ContainerService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+
+	name := d.KeyColumnQuals["name"].GetStringValue()
+	location := d.KeyColumnQuals["location"].GetStringValue()
+
+	resp, err := service.Projects.Locations.Clusters.Get("projects/" + project + "/locations/" + location + "/clusters/" + name).Do()
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+//// TRANSFORM FUNCTION
+
+func gcpKubernetesClusterTurbotData(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("gcpKubernetesClusterTurbotData")
+	cluster := d.HydrateItem.(*container.Cluster)
+
+	splitName := strings.Split(cluster.SelfLink, "/")
+	akas := []string{strings.Replace(cluster.SelfLink, "https", "gcp", 1)}
```

Comment:
@rajeshbal65 Can we do a more specific match i.e replace `https://` with `gcp://`
```suggestion
	akas := []string{strings.Replace(cluster.SelfLink, "https", "gcp", 1)}
```